### ISSUE TITLE
trained model

### ISSUE THREAD

**Issue by missbook520** [Justin900429/PyTorch-DSSN-MER#1]
Can you provide a trained model? Thank you very much！！

**Comment by Justin900429** [comment]
Sorry, the pretrained weights were discarded...

**Comment by missbook520** [comment]
Thank you very much for your reply! I found during training that the loss function is nan, using the casme II dataset, and the training instruction is  python train.py --path data\CASME2-coding-20140508.xlsx --pre CASME_preprocess_before --catego CASME --image_mode F G --weight_save_path CASME_DSSN_weight --model DSSN
I adjusted the batchsize and learning rate, but the loss function is still nan. Can you help me? thank you
![1645442271(1)](https://user-images.githubusercontent.com/48777650/154945180-b2c5ed67-c5ea-44b0-b953-208591842fe1.png)

**Comment by missbook520** [comment]
Thank you very much for your reply! I found during training that the loss function is nan, using the casme II dataset, and the training instruction is  python train.py --path data\CASME2-coding-20140508.xlsx --pre CASME_preprocess_before --catego CASME --image_mode F G --weight_save_path CASME_DSSN_weight --model DSSN
I adjusted the batchsize and learning rate, but the loss function is still nan. Can you help me? thank you
![1645442271(1)](https://user-images.githubusercontent.com/48777650/154945180-b2c5ed67-c5ea-44b0-b953-208591842fe1.png)

**Comment by Justin900429** [comment]
Hi, I released the new preprocess code (not tested, bugs may exist).
You can try the new preprocessing and see the result.
Not sure it will happen again or not :(

**Comment by Justin900429** [comment]
Hi @missbook520, this problem is also discussed in #2.
I will close this issue and switch to #2.
You can leave your concerns in #2 :)
